src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int BOUND = 100;
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
ColorSupplier should be class level variable

Reply:
Are you sure there?

I've tested and it outputted me every time same object.

Reply:
Same with number

Reply:
I'm 100% sure about it

Reply:
you need to change ColorSupplier as I proposal